src/lib/Components/Artist/ArtistArtworks/ArtistCollectionsRailRenderer.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from "@artsy/palette"
+import { ArtistArtworks_artist } from "__generated__/ArtistArtworks_artist.graphql"
+import { ArtistCollectionsRailRendererQuery } from "__generated__/ArtistCollectionsRailRendererQuery.graphql"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import renderWithLoadProgress from "lib/utils/renderWithLoadProgress"
+import React from "react"
+import { graphql, QueryRenderer } from "react-relay"
+import { ArtistCollectionsRailFragmentContainer as ArtistCollectionsRail } from "./ArtistCollectionsRail"
```

Comment:
```suggestion
import { ArtistCollectionsRailFragmentContainer } from "./ArtistCollectionsRail"
```

renaming fragment containers in import statements has been a fairly common pattern at Artsy but going forward we're moving away from it so that components have consistent naming across different files.